docs/gitlab/02-activation.md

```diff
@@ -46,9 +46,10 @@ All you need is [docker](https://www.docker.com/) installed on your machine.
 
    ```bash
    xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' \
-   /opt/Unity/Editor/Unity \
+   /opt/unity/Editor/Unity \
```

Comment:
We should be using alias here

Reply:
full path: `/usr/bin/unity-editor`
so: `unity-editor` should be sufficient
source: https://github.com/game-ci/docker/blob/47c7a39d8989e6c0b501405e7b9c63d3942f63e1/editor/Dockerfile#L174